### PR TITLE
[KYUUBI #1504] Support to capture console out/error into interpreter output for ExecuteScala

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

Redirect console output and error when interpreting scala code, see details in #1504

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
please create a subtask for #1489

**Comment by turboFei** [comment]
> please create a subtask for #1489

ok

File: kyuubi-common/src/test/scala/org/apache/kyuubi/operation/SparkQueryTests.scala [hunk 1]
```diff
@@ -518,4 +518,21 @@ trait SparkQueryTests extends HiveJDBCTestHelper {
       assert(e.getMessage contains "not found: value y")
     }
   }
+
+  test("scala code with console output") {
+    withJdbcStatement() { statement =>
+      statement.execute("SET kyuubi.operation.language=scala")
+      val code =
+        """
+          |spark.sql("set").show(false)
```
Comment (yaooqinn) [inline kyuubi-common/src/test/scala/org/apache/kyuubi/operation/SparkQueryTests.scala:527]: how about `SET kyuubi.operation.language`?

File: kyuubi-common/src/test/scala/org/apache/kyuubi/operation/SparkQueryTests.scala [hunk 2]
```diff
@@ -518,4 +518,21 @@ trait SparkQueryTests extends HiveJDBCTestHelper {
       assert(e.getMessage contains "not found: value y")
     }
   }
+
+  test("scala code with console output") {
+    withJdbcStatement() { statement =>
+      statement.execute("SET kyuubi.operation.language=scala")
+      val code =
+        """
+          |spark.sql("SET kyuubi.operation.language=scala").show(false)
```
Comment (yaooqinn) [inline kyuubi-common/src/test/scala/org/apache/kyuubi/operation/SparkQueryTests.scala:527]: val code = "spark.sql("SET kyuubi.operation.language").show(false)"

**Review by yaooqinn — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merged to master

**Pull request closed without merging**